language-support/ts/daml-ledger/index.ts

```diff
@@ -154,12 +154,12 @@ class Ledger {
   /**
    * Create a contract for a given template.
    */
-  async create<T extends object, K>(template: Template<T, K>, argument: T): Promise<CreateEvent<T, K>> {
-    const payload = {
+  async create<T extends object, K>(template: Template<T, K>, contractPayload: T): Promise<CreateEvent<T, K>> {
+    const command = {
       templateId: template.templateId,
-      argument,
+      payload: contractPayload,
```

Comment:
```suggestion
      payload,
```

Reply:
I guess, it is just in a few other places, @hurryabit refers to the entire command as a payload.

Reply:
Fair, well, whatever you think is best.